music21/analysis/neoRiemannian.py

```diff
@@ -5,8 +5,9 @@
 #
 # Authors:      Maura Church
 #               Michael Scott Cuthbert
+#               Mark Gotham
 #
-# Copyright:    Copyright © 2012 Michael Scott Cuthbert and the music21 Project
+# Copyright:    Copyright © 2017 Michael Scott Cuthbert and the music21 Project
```

Comment:
change to 2012, 2017 -- US law -- we need to put in the start and end date of modifications